docs/authentication/social-connections/linkedin.mdx

```diff
@@ -43,4 +45,4 @@ The last thing you have to do is to enable the _Sign in with LinkedIn_ product f
 
 Copy the **Application ID** and **Secret** from the previous screen. Go back to the Clerk Dashboard and paste them into the respective fields.
 
-Don't forget to click **Apply** in the Clerk dashboard. Congratulations! Social connection with LinkedIn is now configured for your instance.
+Finally, click **Add connection** so that the settings are applied. Congratulations! Social connection with LinkedIn is now configured for your instance.
```

Comment:
```suggestion
Finally, select **Add connection** so that the settings are applied. Congratulations! Social connection with LinkedIn is now configured for your instance.
```